newdle/client/src/components/answer/AnswerPage.js

```diff
@@ -223,22 +226,32 @@ export default function AnswerPage() {
               <em>No options chosen</em>
             )}
           </span>
-          <Button
-            size="large"
-            color={participantHasAnswers ? 'teal' : 'violet'}
-            content={participantHasAnswers ? 'Update your answer' : 'Send your answer'}
-            disabled={
-              submitting ||
-              !canSubmit ||
-              (participantCode && !participant) ||
-              (participantHasAnswers && !participantAnswersChanged)
-            }
-            loading={submitting}
-            icon="send"
-            onClick={() => {
-              answerNewdle();
-            }}
-          />
+          <Input
+            type="text"
+            placeholder="Leave a comment..."
+            className={styles['comment-submit']}
+            value={comment || ''}
+            onChange={(__, {value}) => setComment(value)}
+            action={!isSmallScreen}
+          >
+            <input />
+            <Button
+              size="large"
+              color={participantHasAnswers ? 'teal' : 'violet'}
+              content={participantHasAnswers ? 'Update your answer' : 'Send your answer'}
+              disabled={
+                submitting ||
+                !canSubmit ||
+                (participantCode && !participant) ||
+                (participantHasAnswers && !participantAnswersChanged && _comment === null)
```

Comment:
This will keep the button enabled if you change the comment and then revert that change. Better check if the comment is not equal to the stored comment.

Reply:
Good idea. Since the condition on the top was simplified, this can be enhanced.